com/metacube/SCF/Assignment3/StringOperations.java

```diff
@@ -0,0 +1,112 @@
+package GET2018.com.metacube.SCF.Assignment3;
+
+/**
+ * Copyright (c) 2018 Metacube.com. All rights reserved.
+ * This class accepts string input and perform required operations.
+ * @author Chirag Jain
+ */
+public class StringOperations {
+	/**
+	 * This method reverse given string
+	 * @param str is string passed by user
+	 * @return string after reversing it
+	 */
+	public String toReverse(String str) {	
+		if(str == null) {
+			throw new AssertionError("String is null.");
+		}
+		String rev = "";
```

Comment:
Add check for blank string also. This is applicable throughout the class.

Reply:
In case of blank string it returns blank string which can be considered correct output in all functions